scripts/exrrfs_fcst.sh

```diff
@@ -28,14 +28,19 @@ else
   ${cpreq} ${COMINrrfs}/${RUN}.${PDY}/${cyc}${ensindexstr}/da/restart.${timestr}.nc .
   do_restart='true'
 fi
-offset=$((10#${cyc}%6))
-CDATElbc=$($NDATE -${offset} ${CDATE})
+
+
+#offset=$((10#${cyc}%6))
+#CDATElbc=$($NDATE -${offset} ${CDATE})
+CDATElbc=${CDATE}
 ${cpreq} ${COMINrrfs}/${RUN}.${CDATElbc:0:8}/${CDATElbc:8:2}${ensindexstr}/lbc/lbc*.nc .
+
+
 ${cpreq} ${FIXrrfs}/physics/${PHYSICS_SUITE}/* .
 ln -snf VEGPARM.TBL.fcst VEGPARM.TBL #gge.debug temp
```

Comment:
FYI, with the update to mpas.v8.2.2 and mpasjedi v.3.0, the VEGPARM.TBL files are now the same for the model and the JEDI, so we don't need to keep different versions. We can leave this for a future PR.

Reply:
I can correct this one as David has question on this.